utils/Cargo.toml

```diff
@@ -23,6 +23,7 @@ zeroize.workspace = true
 test-utils = { path = "../test-utils" }
 
 criterion.workspace = true
+rand.workspace = true
```

Comment:
We do not use rand from anywhere except from crypto library. This makes audits much easier.

Reply:
> We do not use rand from anywhere except from crypto library. This makes audits much easier.

Does this apply to dev-dependencies too? As I can see, rand is sometimes used as a dev dependency currently.

Reply:
Nowhere. All randomness should come from the crypto library.